manifests/noderesourcetopology/scheduler-config.yaml

```diff
@@ -19,3 +22,11 @@ profiles:
       kubeconfigpath: "/etc/kubernetes/scheduler.conf"
       namespaces:
       - "test-namespace"
+  - name: NodeResourceTopologyResourceAllocationScore
```

Comment:
Again, we can one plugin to serve as both filter and score plugins, and if so the args can be merged into NodeResourceTopologyMatchArgs.